.taskcluster.yml

```diff
@@ -245,28 +262,30 @@ tasks:
             - --login
             - -cx
             - >-
-              git fetch origin
-              && git reset --hard origin/master
+              cd ..
+              && git clone ${repository} repository
+              && cd repository
               && python tools/taskcluster/release.py \
-                --track nightly \
+                --track ${track} \
                 --commit \
-                --output /opt/focus-android/app/build/outputs/apk \
+                --output /opt/repository/app/build/outputs/apk \
```

Comment:
I'm surprised changing this folder doesn't require a new docker image, but I confess I didn't look much into it :)

Reply:
I also changed the line above: https://github.com/mozilla-mobile/focus-android/pull/4129/files/c081985cdf6c936d2083afce81ac2afd61ed52b6#diff-ac0229d1171c0983b27003af4c7441a5R266